sql/sql_yacc.yy

```diff
@@ -2982,6 +2982,12 @@ opt_ev_status:
             Lex->event_parse_data->status_changed= true;
             $$= 1;
           }
+        | ENABLE_SYM ON SLAVE
+          {
+            Lex->event_parse_data->status= Event_parse_data::SLAVESIDE_ENABLED;
```

Comment:
`status` would be set on slave straight to `ENABLE` unless the event is CREATE-ed with `DISABLE` status. 
Logics inside `Event_parse_data::check_originator_id()` would need some adjustment, which is to disable the event on slave 
```
status= Event_parse_data::DISABLED
```
without the new **logging** option.

The status of event on any server then would flip between just two `ENABLE` and `DISABLE`.
I think the slave should preserve the new option in its binlog when one is configured on the slave.